tests/ci_build/print_r_stacktrace.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
-
```

Comment:
Do we need this script for action?

Reply:
Good point. Let me put it back and update the action yml.